stable/yugaware/templates/statefulset.yaml

```diff
@@ -538,6 +538,23 @@ spec:
           ports:
             - containerPort: 9000
               name: yugaware
+          {{- if .Values.yugaware.pod.probes.enabled }}
+          startupProbe:
+            failureThreshold: {{ div .Values.yugaware.pod.probes.startupTimeSec.max (div .Values.yugaware.pod.probes.startupTimeSec.min 2) }}
+            initialDelaySeconds: {{ .Values.yugaware.pod.probes.startupTimeSec.min }}
+            periodSeconds: {{ div .Values.yugaware.pod.probes.startupTimeSec.min 2 }}
+            timeoutSeconds: {{ div .Values.yugaware.pod.probes.startupTimeSec.min 4 }}
+            httpGet:
+                path: /api/app_version
+                port: yugaware
+          livenessProbe:
+            failureThreshold: {{ .Values.yugaware.pod.probes.livenessProbe.failureThreshold }}
+            periodSeconds: {{ .Values.yugaware.pod.probes.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ div .Values.yugaware.pod.probes.livenessProbe.periodSeconds 2 }}
```

Comment:
is this a connection timeout for the probe ? 
Can you describe why it should be 0.5*periodSeconds?

Reply:
Disclaimer: I didn't check how k8s probes works exactly.
Timeout affects probes period, as it reflectx maximum time a probe could take.
If a probe time is taken into account to ensure period is static, it should be less than period to avoid accumulating probes. Divisinon by 2 looks fine for this case and gives some space for other hiccups.
In case a probe time isn't counted as a part of period, it effectively extends period. Extending by 50% is something this workload could live with.